packages/samples/specs/resource-manager/legacy/legacy-operations/main.tsp

```diff
@@ -0,0 +1,63 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using Http;
+using Rest;
+using Versioning;
+using Azure.ResourceManager;
+
+/** Contoso Resource Provider management API. */
+#suppress "@azure-tools/typespec-azure-resource-manager/missing-operations-endpoint" ""
+@armProviderNamespace
+@service(#{ title: "ContosoProviderHubClient" })
+@versioned(Versions)
+namespace Microsoft.ContosoProviderHub;
+
+/** Contoso API versions */
+enum Versions {
+  /** 2021-10-01-preview version */
+  @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+  @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v5)
+  `2021-10-01-preview`,
+}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" ""
+#suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model" ""
+model BestPractice is Azure.ResourceManager.ProxyResource<{}> {
+  ...ResourceNameParameter<
+    Resource = BestPractice,
+    KeyName = "bestPracticeName",
+    SegmentName = "bestPractices",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface BestPracticesVersionOps
```

Comment:
Alias = I think is the correct way

Reply:
This shouldn't matter as there are no concrete operations, I think.

Reply:
Yeah, I was thinking that as I wrote this - we should probably make this the norm when using the legacy template (and when using the ResourceOperations template in azure.core)